### PR TITLE
Admin entruempelung

### PR DESCRIPTION
**Pull request by moonglum**
* [x] Remove dashboard and tracking. Nobody use it, why then collect data and pull in lots and lots of code?
* [x] Clean up the admin JavaScript...

I wanted to remove the need for CoffeeScript in the backoffice (the frontoffice JS will be rewritten, and I definitely will not use CoffeeScript). Most JS could be removed anyway, only one file was left. I inserted it compiled.

### CODE REVIEW

File: app/assets/javascripts/active_admin.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  if ($(".admin_events.edit,.admin_events.new").length > 0) {
+    this.rules = [];
+    this.exdates = [];
+    dayRules = {
+      1: "ersten",
+      2: "zweiten",
+      3: "dritten",
+      4: "vierten",
+      5: "fünften",
+      "-1": "letzten"
```
Comment (bascht) [inline app/assets/javascripts/active_admin.js]: nice

Comment (moonglum) [inline app/assets/javascripts/active_admin.js]: This entire script is a **gem**

**Comment by bascht** [comment]
Me likey.

I've created a quick backup, in case shit goes south.
```
$ ls -lh 2017-02-27-0117-hacken-in-production.sql 
-rw-r--r--. 1 hacken hacken 46M Feb 27 01:17 2017-02-27-0117-hacken-in-production.sql
```

**Comment by bascht** [comment]
:100:

**Comment by bascht** [comment]
screenshot_20170227_012102
sweet.

**Pull request merged**